packages/admin-panel/src/api/mutations/useRequestResetPassword.js

```diff
@@ -8,7 +8,7 @@ import { post } from '../../VizBuilderApp/api';
 
 export const useRequestResetPassword = () => {
   return useMutation(({ emailAddress }) => {
-    return post('auth/resetPassword', {
+    return post('requestResetPassword', {
```

Comment:
Could we also update this in `datatrak-web` and `tupaia-web`? We can also remove `requestResetPassword.js` from `web-config-server` and the line in `tupaia-web-server` that forwards `resetPassword` to the `web-config-server` :pray: